libs/beacon/include/beacon/beacon_manager.hpp

```diff
@@ -158,6 +163,51 @@ class BeaconManager
 
 namespace serializers {
 
+template <typename D>
```

Comment:
serialize the absolute neccessary stuff to recover the beacon manager (not the certificate though which is set after). I don't think we want to serialize the certificate ever.

Reply:
I am confused about this not saving of the certificate at all. This certificate is tied to them being a member of the cabinet since the cabinet consists of the public keys of all people who won the auction. All messages sent during the DKG and entropy signature shares need to be signed with this certificate so that other DKG members know it is genuinely from them and worth processing.

Reply:
Also, cabinet size and cabinet index are deducible from identity_to_index if you know your own public key from certificate (which you will need as mentioned above)